doc/releases/changelog-dev.md

```diff
@@ -47,6 +47,28 @@ keyword argument when using `GellMann`, which determines which of the 8 Gell-Man
         1: ─╰RXY(1.570796+0.00j)─╰RYX(1.570796+0.00j)─┤ ╰<Z@Z>
   ```
 
+* The `QNode` class now accepts an ``auto`` interface, which automatically detects the interface
+  of the given input.
+  [(#3132)](https://github.com/PennyLaneAI/pennylane/pull/3132)
+
+  ```pycon
+  >>> dev = qml.device("default.qubit", wires=2)
+  >>> @qml.qnode(dev, interface="auto")
+      def circuit(weight):
+          qml.RX(weight[0], wires=0)
+          qml.RY(weight[1], wires=1)
+          return qml.expval(qml.PauliZ(0))
```

Comment:
The formatting is a bit off here, lines of code after lines starting with `>>>` should start with `...`

![image](https://user-images.githubusercontent.com/24476053/195947984-dcb00dec-b811-42c0-a92f-311bba0f36c0.png)

An alternative is to first have a python block and then a pycon block.

Reply:
Added python block